src/pages/Signup.jsx

```diff
@@ -0,0 +1,300 @@
+// React Imports
+import React, { useState, useEffect } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { TextField, Button, CardHeader } from '@mui/material';
+
+import {
+  createAcl,
+  createContainerAt,
+  getResourceAcl,
+  getSolidDatasetWithAcl,
+  getThing,
+  getWebIdDataset,
+  setAgentDefaultAccess,
+  setAgentResourceAccess,
+  saveAclFor,
+  getStringNoLocale
+} from '@inrupt/solid-client';
+import { FOAF } from '@inrupt/vocab-common-rdf';
+
+import { ENV } from '@constants';
+
+import Box from '@mui/material/Box';
+import Card from '@mui/material/Card';
+import Container from '@mui/material/Container';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+import useContactsList from '@hooks/useContactsList';
+import useSession from '@hooks/useSession';
+
+const formRowStyle = {
+  margin: '20px 0'
+};
+
```

Comment:
I assume we'll focus on fully applying MUI styling in another PR, so I don't think it's essential for this one

Reply:
Yeah I don't have any design from UX for this, so I just wanted to get something clean and readable